core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/TagsImpl.java

```diff
@@ -88,10 +91,19 @@
         final String[] tagIds = asset.getProperties().get(tagPropertyName, new String[]{});
 
         for (final String tagId : tagIds) {
-            final Tag tag = tagManager.resolve(tagId);
-
-            if (tag != null) {
-                overrideTagTitles.add(tag.getTitle(locale));
+        	final Tag tag = tagManager.resolve(tagId);
```

Comment:
This method is getting confusing :) .. i wonder if we should take this as opportunity to refactor it..

1) If no properties are selected, then use tagTitles (existing default behavior)
2) Update dialog to write multiple values to tagPropertyName
3) For each of those defined properties, do a `asset.getProperties().get(..)`  ... (which has the problem we dont actually know the return types... :() 
4) for each result from each tagPropertyName, check if the value resolves as a tagId (if it does, assume its a real tag) and get the localized title
5) If tag == null, then assume its a title, and display that (not sure how to best localize this; i18n?)
6) dump all the titles them all into 1 list that is maybe sortable (alpha a-z?)